spec/sonar/cli_spec.rb

```diff
@@ -82,6 +82,22 @@
         end
       end
     end
+
+    describe 'search_all command' do
```

Comment:
These specs are rather weak due to the level of stubbing.  At a high level all I want to test is the IP vs. domain input functionality.  Everything else is handled by the `search` method in the CLI class.  I was thinking to use `expect(Sonar::CLI).to receive(:search).with(:fdns, 'rapid7.com')` (when domain given) but unfortunately I couldn't get it to work.